transport_plugins/jlink/iotile_transport_jlink/jlink_background.py

```diff
@@ -161,8 +168,118 @@ def _trigger_rpc(self, device_info):
         else:
             raise HardwareError("Unknown RPC trigger method", method=method)
 
-    def _dump_all_ram(self, device_info, _control_info, _args, _progress_callback):
-        memory = self._read_memory(device_info.ram_start, device_info.ram_size)
+    def _continue(self):
+        self._jlink.step()
+        self._jlink._dll.JLINKARM_Go()
+        while not self._jlink.halted():
+            pass
```

Comment:
You are going to want to put a `time.sleep(0.01)` line here so that the CPU core of this thread doesn't peg while the mcu is running.

Reply:
Fixed.